packages/flutter/lib/src/material/expansion_panel.dart

```diff
@@ -95,6 +96,11 @@ class ExpansionPanel {
   /// Defaults to false.
   final bool isExpanded;
 
+  /// Defines the splash color of the panel if [canTapOnHeader] is true, or the expand/collapse IconButton if [canTapOnHeader] is false.
```

Comment:
```suggestion
  /// Defines the splash color of the panel if [canTapOnHeader] is true, or the splash color of the expand/collapse IconButton if [canTapOnHeader] is false.
```

Reply:
It's a bit hard to understand the reasoning at first glance though. Maybe:
```suggestion
	  /// Defines the splash color of the expand/collapse icon button, and the splash color of the panel if [canTapOnHeader] is true.
```